assets/css/style.css

```diff
@@ -16597,6 +16597,9 @@ form.civiremote-event-register-form .form-item[data-drupal-selector=edit-confirm
 #funding-form .json-forms-array-wrapper > .form-item {
   margin-top: 0;
 }
+#funding-form .custom-control {
+  z-index: initial !important;
```

Comment:
Is there really no way to avoid `!important`?

Reply:
It's not applied without `!important`.

Reply:
As discussed, let's use a more specific selector instead of `!important`.

Reply:
Changed.